redislock.go

```diff
@@ -165,10 +204,11 @@ func (l *Lock) TTL(ctx context.Context) (time.Duration, error) {
 // May return ErrNotObtained if refresh is unsuccessful.
 func (l *Lock) Refresh(ctx context.Context, ttl time.Duration, opt *Options) error {
 	ttlVal := strconv.FormatInt(int64(ttl/time.Millisecond), 10)
-	status, err := luaRefresh.Run(ctx, l.client, []string{l.key}, l.value, ttlVal).Result()
+	res, err := luaRefresh.Run(ctx, l.client, l.keys, l.value, ttlVal).Result()
 	if err != nil {
 		return err
-	} else if status == int64(1) {
+	}
+	if refreshCount := res.(int64); refreshCount == int64(len(l.keys)) {
```

Comment:
Same "script could return boolean" note here, I guess.

Doesn't prevent us from merging and then polishing it ourselves.

Reply:
Same as above.